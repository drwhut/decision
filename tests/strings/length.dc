Start~#1

> The empty string
Length("")~#2
Print(#1, #2)~#3

> A normal string
Length("Hello, world!")~#4
Print(#3, #4)~#5

> A string with escape characters in
Length("Hello\tthere\n")~#6
Print(#5, #6)~#7

> A variable string
[Variable(myStr, String, "boop")]
myStr~#8
Length(#8)~#9
Print(#7, #9)~#10