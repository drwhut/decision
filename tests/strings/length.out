0
13
12
4
